Intersect.Client/Entities/Player.cs

```diff
@@ -1049,10 +1066,14 @@ public void TryUseSpell(int index)
 
         public long GetSpellCooldown(Guid id)
         {
-            if (SpellCooldowns.ContainsKey(id))
+            if (SpellCooldowns.ContainsKey(id) && SpellCooldowns[id] > Timing.Global.Milliseconds)
             {
                 return SpellCooldowns[id];
             }
+            else if (!SpellBase.Get(id).IgnoreGlobalCooldown && Globals.Me.GlobalCooldown > Timing.Global.Milliseconds)
```

Comment:
Also use TryGet here, Get can return null and we'd have a null pointer exception here